Containerfile

```diff
@@ -13,6 +13,9 @@ RUN go mod download
 COPY main.go main.go
 #COPY api/ api/
 COPY controllers/ controllers/
+COPY controllers/constants/ovms-metrics.json metrics_dashboards/ovms-metrics.json
+COPY controllers/constants/tgis-metrics.json metrics_dashboards/tgis-metrics.json
+COPY controllers/constants/vllm-metrics.json metrics_dashboards/vllm-metrics.json
```

Comment:
where are these files?

Reply:
They're from Killian's [PR 209](https://github.com/opendatahub-io/odh-model-controller/pull/209)

Reply:
And just now added (forgot to put them in), but the caikit-metrics.json file is from [PR 218](https://github.com/opendatahub-io/odh-model-controller/pull/218)

Reply:
okay, @KillianGolds's PR is merged, can you please rebase yours to use the files from there?